ironfish-cli/src/commands/start.ts

```diff
@@ -194,7 +220,7 @@ export default class Start extends IronfishCommand {
     this.log(`Peer Identity ${node.peerNetwork.localPeer.publicIdentity}`)
     this.log(`Peer Agent    ${node.peerNetwork.localPeer.agent}`)
     this.log(`Peer Port     ${peerPort}`)
-    this.log(`Bootstrap     ${bootstraps.join(',') || 'NONE'}`)
+    this.log(`Bootstrap     ${bootstraps.join(',') || 'FROM NETWORK DEFINITION'}`)
```

Comment:
Oh, are the bootstrap nodes not loaded at this point? Or could we pull them from the node somehow?